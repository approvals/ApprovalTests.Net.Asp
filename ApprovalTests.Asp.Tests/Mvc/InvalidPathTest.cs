using ApprovalTests.Asp.Mvc;
using ApprovalTests.Reporters;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ApprovalTests.Asp.Tests.Mvc
{
    [TestClass]
    public class InvalidPathTest
    {
        //TODO: We need to work on this
        [TestMethod]
        public void TestErrorMessage()
        {
            MvcApprovals.VerifyMvcPage<TestableWithInvalidPathController>(t => t.IndexTest);
        }
    }
}
