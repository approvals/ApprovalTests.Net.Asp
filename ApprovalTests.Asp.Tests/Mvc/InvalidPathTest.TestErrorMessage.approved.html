<pre>
You MVC Project is missing a dll test project referencing
Could not load file or assembly 'SmallFry, Version=0.5.0.0, Culture=neutral, PublicKeyToken=a0877c8a8e3dbcef' or one of its dependencies. The system cannot find the file specified.
</pre>